CHANGES.md

```diff
@@ -140,6 +140,9 @@ Vernacular commands
 
 - `Hypotheses` and `Variables` can now take implicit binders inside sections.
 
+- `Coercion` does not warn ambiguous paths which are obviously convertible with
```

Comment:
I'm afraid people will wonder what "obviously convertible" means. Should it just be "convertible"?

Reply:
Some inheritance paths which do not respect the uniform inheritance condition may be reported as ambiguous paths even if they are convertible with existing ones.

Reply:
I have provided a better description of this change in #10322.